app/controllers/IndexController.scala

```diff
@@ -43,21 +45,18 @@ class IndexController @Inject()(
         Spoke(charityInfoRoutes.CharityNameController.onPageLoad(NormalMode).url, getStatus(userAnswers.get(Section1Page)))))
 
       val section2 = TaskListSection(List(
```

Comment:
Can we updated TaskListSectionas follows to get rid of unwanted Spoke List
TaskListSection(url: String, status: String)